StoreKitTests/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>$(PRODUCT_BUNDLE_PACKAGE_TYPE)</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
+	<string>3.12.0</string>
```

Comment:
Should we add this to the prepare release lane?

Reply:
do you mean `prepare_next_version`, `deploy` or something else? 
for `prepare_next_version`, this will automatically get bumped by `increase_version_number`. 
for `deploy`, we want to just deploy whatever is in the tag

Reply:
I meant `increase_version_number`, if it does it automatically perfect then 👍